File: 15/15.cpp
#include <iostream>
using namespace std;
int main() {
    int num1,num2;
    cout<<"Enter two numbers:";
    cin>>num1>>num2;
    int min = (num1<num2)*num1 + (num2<num1)*num2;
    int max = (num1>num2)*num1 + (num2>num1)*num2;
    cout<<"The maximum number is "<<max<<endl;
    cout<<"The minimum number is "<<min;
    return 0;
}

File: 15/15.py
num1,num2= map(int,input("Enter two numbers:").split())
min = (num1<num2)*num1 + (num2<num1)*num2
max = (num1>num2)*num1 + (num2>num1)*num2
print(f"The maximum number is {max}")
print(f"The minimum number is {min}")
